app/test_split_integer.py

```diff
@@ -2,20 +2,20 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert (sum(split_integer(8, 3)) == 8)
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    assert (split_integer(12, 3) == [4, 4, 4])
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    assert (split_integer(6, 1)[0] == 6)
```

Comment:
Otherwise it won't check the whole list, only first arg
```suggestion
    assert split_integer(6, 1) == [6]
```